src/convertToObject.js

```diff
@@ -3,16 +3,42 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const arrOfProperties = sourceString.split(';');
+
+  for (const index in arrOfProperties) {
+    let elem = arrOfProperties[index];
+
+    while (elem.startsWith('/n')) {
+      elem = elem.slice(0, 2);
+    }
+    arrOfProperties[index] = elem.trim();
+  }
+
+  const validProps = arrOfProperties.filter((prop) => prop.length !== 0);
```

Comment:
you can write more simple like `.filter(prop => prop)` or even do this at the beginning of the loop

```
 for (const prop of properties) {
  if (!prop) {
      continue;
  }
```    
we just skip empty properties